runtime/doc/usr_45.txt

```diff
@@ -234,16 +222,7 @@ Unicode support.  You can do this with Vim, although characters that are not
 supported by the terminal will not be displayed.  The layout of the text
 will be preserved.  >
```

Comment:
what `>` ? :) oh, it was both concealed _and_ invisible... Perhaps the later (fg==bg) should not happen when _editing_ help files.